students/chaycasso/Lesson02/gridprinter.py

```diff
@@ -0,0 +1,79 @@
+#Grid Printer Exercise
+#Chay Casso, 1/29/18
+
+def grid1():
+    """Draw a simple 2x2 grid composed of 5x5 squares."""
+    plus = "+"
+    minus = "-"
+    blank = " "
+    pipe = "|"
```

Comment:
Instead of defining the `plus`, `minus`, `blank`, and `pipe` in every function.  You can define at the top of your code (line 3 or above) in all CAPS and they will be available to all functions